debugger/pythonExtension/gdb_sgx_plugin.py

```diff
@@ -17,7 +17,9 @@
 OE_ENCLAVE_HEADER_FORMAT = 'QQQQQ'
 OE_ENCLAVE_MAGIC_VALUE = 0x20dc98463a5ad8b8
 
-OE_ENCLAVE_FLAGS_OFFSET = 0x588
+# The following is the offset of the 'debug' and
+# 'simulate' fields which must lie one after the other.
```

Comment:
Can you add flags somewhere as the debug and simulate are the flags? How about 'simulate' flags?